2020-03-15 02:38:26 [scrapy] DEBUG: Retrying <GET http://andromeda.unimet.edu.ve/robots.txt> (failed 1 times): DNS lookup failed: no results for hostname lookup: andromeda.unimet.edu.ve.
2020-03-15 02:38:26 [scrapy] DEBUG: Retrying <GET http://andromeda.unimet.edu.ve/robots.txt> (failed 2 times): DNS lookup failed: no results for hostname lookup: andromeda.unimet.edu.ve.
2020-03-15 02:38:26 [scrapy] DEBUG: Gave up retrying <GET http://andromeda.unimet.edu.ve/robots.txt> (failed 3 times): DNS lookup failed: no results for hostname lookup: andromeda.unimet.edu.ve.
2020-03-15 02:38:26 [scrapy] ERROR: Error downloading <GET http://andromeda.unimet.edu.ve/robots.txt>: DNS lookup failed: no results for hostname lookup: andromeda.unimet.edu.ve.
Traceback (most recent call last):
  File "c:\users\jean\appdata\local\programs\python\python38-32\lib\site-packages\twisted\internet\defer.py", line 1416, in _inlineCallbacks
    result = result.throwExceptionIntoGenerator(g)
  File "c:\users\jean\appdata\local\programs\python\python38-32\lib\site-packages\twisted\python\failure.py", line 512, in throwExceptionIntoGenerator
    return g.throw(self.type, self.value, self.tb)
  File "c:\users\jean\appdata\local\programs\python\python38-32\lib\site-packages\scrapy\core\downloader\middleware.py", line 44, in process_request
    defer.returnValue((yield download_func(request=request, spider=spider)))
  File "c:\users\jean\appdata\local\programs\python\python38-32\lib\site-packages\twisted\internet\defer.py", line 654, in _runCallbacks
    current.result = callback(current.result, *args, **kw)
  File "c:\users\jean\appdata\local\programs\python\python38-32\lib\site-packages\twisted\internet\endpoints.py", line 981, in startConnectionAttempts
    raise error.DNSLookupError(
twisted.internet.error.DNSLookupError: DNS lookup failed: no results for hostname lookup: andromeda.unimet.edu.ve.
2020-03-15 02:38:26 [scrapy] DEBUG: Retrying <GET http://andromeda.unimet.edu.ve/catalogo/php1/buscar.php?xx=9999&base=tesis&cipar=tesis.par&Expresion=(romero/(100))&Opcion=detalle&Formato=a&from=&to=> (failed 1 times): DNS lookup failed: no results for hostname lookup: andromeda.unimet.edu.ve.
2020-03-15 02:38:26 [scrapy] DEBUG: Retrying <GET http://andromeda.unimet.edu.ve/catalogo/php1/buscar.php?xx=9999&base=tesis&cipar=tesis.par&Expresion=(romero/(100))&Opcion=detalle&Formato=a&from=&to=> (failed 2 times): DNS lookup failed: no results for hostname lookup: andromeda.unimet.edu.ve.
2020-03-15 02:38:26 [scrapy] DEBUG: Gave up retrying <GET http://andromeda.unimet.edu.ve/catalogo/php1/buscar.php?xx=9999&base=tesis&cipar=tesis.par&Expresion=(romero/(100))&Opcion=detalle&Formato=a&from=&to=> (failed 3 times): DNS lookup failed: no results for hostname lookup: andromeda.unimet.edu.ve.
2020-03-15 02:38:26 [scrapy] ERROR: Error downloading <GET http://andromeda.unimet.edu.ve/catalogo/php1/buscar.php?xx=9999&base=tesis&cipar=tesis.par&Expresion=(romero/(100))&Opcion=detalle&Formato=a&from=&to=>
Traceback (most recent call last):
  File "c:\users\jean\appdata\local\programs\python\python38-32\lib\site-packages\twisted\internet\defer.py", line 1416, in _inlineCallbacks
    result = result.throwExceptionIntoGenerator(g)
  File "c:\users\jean\appdata\local\programs\python\python38-32\lib\site-packages\twisted\python\failure.py", line 512, in throwExceptionIntoGenerator
    return g.throw(self.type, self.value, self.tb)
  File "c:\users\jean\appdata\local\programs\python\python38-32\lib\site-packages\scrapy\core\downloader\middleware.py", line 44, in process_request
    defer.returnValue((yield download_func(request=request, spider=spider)))
  File "c:\users\jean\appdata\local\programs\python\python38-32\lib\site-packages\twisted\internet\defer.py", line 654, in _runCallbacks
    current.result = callback(current.result, *args, **kw)
  File "c:\users\jean\appdata\local\programs\python\python38-32\lib\site-packages\twisted\internet\endpoints.py", line 981, in startConnectionAttempts
    raise error.DNSLookupError(
twisted.internet.error.DNSLookupError: DNS lookup failed: no results for hostname lookup: andromeda.unimet.edu.ve.
2020-03-15 02:38:26 [scrapy] INFO: Closing spider (finished)
2020-03-15 02:38:26 [scrapy] INFO: Dumping Scrapy stats:
{'downloader/exception_count': 6,
 'downloader/exception_type_count/twisted.internet.error.DNSLookupError': 6,
 'downloader/request_bytes': 1719,
 'downloader/request_count': 6,
 'downloader/request_method_count/GET': 6,
 'elapsed_time_seconds': 0.240901,
 'finish_reason': 'finished',
 'finish_time': datetime.datetime(2020, 3, 15, 6, 38, 26, 918020),
 'log_count/DEBUG': 6,
 'log_count/ERROR': 2,
 'log_count/INFO': 7,
 'retry/count': 4,
 'retry/max_reached': 2,
 'retry/reason_count/twisted.internet.error.DNSLookupError': 4,
 "robotstxt/exception_count/<class 'twisted.internet.error.DNSLookupError'>": 1,
 'robotstxt/request_count': 1,
 'scheduler/dequeued': 3,
 'scheduler/dequeued/memory': 3,
 'scheduler/enqueued': 3,
 'scheduler/enqueued/memory': 3,
 'start_time': datetime.datetime(2020, 3, 15, 6, 38, 26, 677119)}
2020-03-15 02:38:26 [scrapy] INFO: Spider closed (finished)
